ReactWindows/ReactNative/ReactPage.cs

```diff
@@ -88,10 +89,20 @@ public virtual string JavaScriptMainModuleName
         /// <param name="arguments">The launch arguments.</param>
         public void OnCreate(string arguments)
         {
+            OnCreate(arguments, null);
```

Comment:
Instead of passing null around, please use a default(JObjext) and check for jObject.HasValues to determine if the props are empty.